combat-trainer.lic

```diff
@@ -1628,12 +1628,11 @@ class SpellProcess
       if data['abbrev'] == 'moonblade'
         game_state.casting_moonblade = true
         last_moon = moon_used_to_summon_weapon
-        data['before'] = [{ 'message' => 'get moon', 'matches' => ['already holding that', 'You grab', 'referring to'] }]
-        if UserVars.moons['visible'].include?(last_moon) && UserVars.moons[last_moon]['timer'] >= 4
+        if DRCMM.hold_moon_weapon? && UserVars.moons['visible'].include?(last_moon) && UserVars.moons[last_moon]['timer'] >= 4
           moon = last_moon
           data['cast'] = "cast #{moon} refresh"
         else
-          data['before'] << { 'message' => 'drop moon', 'matches' => ['open your hand', 'referring to'] }
+          DRCMM.drop_moon_weapon?
```

Comment:
The commands `get moon` and `drop moon` are too ambiguous and can conflict with other "moon" worn items. This change makes makes use of the new DRCMM utilities for holding a moon weapon to refresh it, or dropping what you have to cast a new one, rather than using the `buff` script's "before" hook.